.gitignore

```diff
@@ -1,4 +1,6 @@
 .vscode
 .DS_Store
 .env
-node_modules
+.api
+.database
```

Comment:
Do we need `.api` and `.database` here? I don't believe these get generated

Reply:
We do need both of the above, to avoid being tracked.
Above both are both Docker persistent storage volumes.